src/Pact/Analyze/Model/Text.hs

```diff
@@ -9,26 +9,32 @@ module Pact.Analyze.Model.Text
   ( showModel
   ) where
 
-import           Control.Lens             (Lens', at, ifoldr, (^.))
-import qualified Data.Foldable            as Foldable
-import           Data.Map.Strict          (Map)
-import qualified Data.Map.Strict          as Map
-import           Data.Monoid              ((<>))
-import           Data.SBV                 (SBV, SymWord)
-import qualified Data.SBV                 as SBV
-import qualified Data.SBV.Internals       as SBVI
-import           Data.Text                (Text)
-import qualified Data.Text                as T
-
-import qualified Pact.Types.Info          as Pact
-import           Pact.Types.Runtime       (tShow)
-
-import           Pact.Analyze.Model.Graph (linearize)
+import           Control.Lens               (Lens', at, ifoldr, view, (^.))
+import           Control.Monad.State.Strict (State, evalState, get, modify)
+import qualified Data.Foldable              as Foldable
+import           Data.Map.Strict            (Map)
+import qualified Data.Map.Strict            as Map
+import           Data.Monoid                ((<>))
+import           Data.SBV                   (SBV, SymWord)
+import qualified Data.SBV                   as SBV
+import qualified Data.SBV.Internals         as SBVI
+import           Data.Text                  (Text)
+import qualified Data.Text                  as T
+import           GHC.Natural                (Natural)
+
+import qualified Pact.Types.Info            as Pact
+import           Pact.Types.Runtime         (tShow)
+
+import           Pact.Analyze.Model.Graph   (linearize)
 import           Pact.Analyze.Types
 
 indent :: Text -> Text
 indent = ("  " <>)
 
+indent' :: Natural -> Text -> Text
```

Comment:
perhaps `indent` and `indent'` should swap names

Reply:
Will do